components/TextFields/src/MDCTextInputControllerFilled.m

```diff
@@ -48,7 +48,7 @@
 static const CGFloat MDCTextInputControllerFilledThreeQuartersPadding = 12;
 
 static inline UIColor *MDCTextInputControllerFilledDefaultBorderFillColorDefault() {
-  return [UIColor colorWithWhite:0 alpha:.06f];
+  return [UIColor colorWithWhite:0 alpha:(CGFloat).06];
```

Comment:
tip: for consistency, let's make it `(CGFloat)0.06`

Reply:
Done throughout.